shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -98,7 +101,11 @@ void FlutterTizenView::CreateRenderSurface(
                                          window->GetRenderTargetDisplay(),
                                          geometry.width, geometry.height);
     } else {
+#ifdef NUI_SUPPORT
+      auto* tizen_view = reinterpret_cast<TizenViewNui*>(tizen_view_.get());
+#else
       auto* tizen_view = reinterpret_cast<TizenView*>(tizen_view_.get());
+#endif
```

Comment:
Isn't `TizenViewNui` already a subtype of `TizenView`? Is the change necessary?

Reply:
I fixed